store/conf/attrs/zimbra-attrs.xml

```diff
@@ -9661,4 +9661,8 @@ TODO: delete them permanently from here
   <desc>This attribute contains the number of consecutive failures before requiring authentication via CAPTCHA</desc>
 </attr>
 
+<attr id="5007" name="zimbraEmailPrintoutHeader" type="boolean" cardinality="single" optionalIn="account,cos" flags="accountInfo,accountInherited" since="8.8.6">
```

Comment:
Is accountInfo necessary?

Reply:
I had to add accountinfo so that I can access zimbraEmailPrintoutHeaderEnabled attribute in zm-mailbox using mailbox.attrs.zimbraEmailPrintoutHeaderEnabled